Ejercicios/Clase4/index.html

```diff
@@ -0,0 +1,503 @@
+    Curabitur at sollicitudin metus.
+    Pellentesque finibus elit a condimentum facilisis. Cras fringilla facilisis mi sed feugiat. 
+    Donec in vestibulum arcu, nec gravida tellus. Praesent ut placerat urna. Curabitur vel ipsum erat. 
+    Aenean eget tellus ultricies, viverra est ac, dignissim eros. Suspendisse finibus lobortis aliquet. 
+    Aliquam nec ipsum at ex ullamcorper congue sed eu leo. Donec accumsan aliquet dignissim. 
+    Nam et purus lobortis leo eleifend venenatis et sit amet felis. Proin suscipit ex tellus, et dignissim justo ultricies sit amet. 
+    Etiam semper urna vel vehicula euismod.
+    Proin libero augue, porta ut vestibulum et, cursus ac velit. 
+    Mauris pharetra eleifend imperdiet. Nullam justo odio, sodales at lobortis at, elementum ac nunc. 
+    Integer ut commodo nibh. Phasellus pharetra placerat sodales. Suspendisse potenti. 
+    Cras lobortis mauris non felis cursus, a ullamcorper sem viverra. Donec pharetra lacus quis quam ornare, quis viverra nisi hendrerit. 
+    Nunc consequat lacinia nunc a luctus. Donec volutpat pellentesque dictum. Phasellus maximus facilisis augue sed fringilla.
+    Fusce sodales in ex nec vulputate. Suspendisse a magna elit. Mauris scelerisque ante auctor ipsum interdum, mollis ultrices massa posuere. 
+    Duis quis tempus nibh, vel faucibus orci. Vestibulum vel neque a enim lacinia sollicitudin suscipit nec libero. Vestibulum ut pretium enim. 
+    Curabitur at sollicitudin metus.
+    Pellentesque finibus elit a condimentum facilisis. Cras fringilla facilisis mi sed feugiat. 
+    Donec in vestibulum arcu, nec gravida tellus. Praesent ut placerat urna. Curabitur vel ipsum erat. 
+    Aenean eget tellus ultricies, viverra est ac, dignissim eros. Suspendisse finibus lobortis aliquet. 
+    Aliquam nec ipsum at ex ullamcorper congue sed eu leo. Donec accumsan aliquet dignissim. 
+    Nam et purus lobortis leo eleifend venenatis et sit amet felis. Proin suscipit ex tellus, et dignissim justo ultricies sit amet. 
+    Etiam semper urna vel vehicula euismod.
+    Proin libero augue, porta ut vestibulum et, cursus ac velit. 
+    Mauris pharetra eleifend imperdiet. Nullam justo odio, sodales at lobortis at, elementum ac nunc. 
+    Integer ut commodo nibh. Phasellus pharetra placerat sodales. Suspendisse potenti. 
+    Cras lobortis mauris non felis cursus, a ullamcorper sem viverra. Donec pharetra lacus quis quam ornare, quis viverra nisi hendrerit. 
+    Nunc consequat lacinia nunc a luctus. Donec volutpat pellentesque dictum. Phasellus maximus facilisis augue sed fringilla.
+    Fusce sodales in ex nec vulputate. Suspendisse a magna elit. Mauris scelerisque ante auctor ipsum interdum, mollis ultrices massa posuere. 
+    Duis quis tempus nibh, vel faucibus orci. Vestibulum vel neque a enim lacinia sollicitudin suscipit nec libero. Vestibulum ut pretium enim. 
+    Curabitur at sollicitudin metus.
+    </p>
+    <h2 id="pintu">Pinturitas</h2>
+    <p>
```

Comment:
Es demasiado largo este parrafo! se te complicaria manejarlo en un futuro

Reply:
i edited this paragraphs they are more small, i'm responding you in english